Behavior/BehaviorDelegate.swift

```diff
@@ -21,6 +21,19 @@ import SwiftUI
 class BehaviorDelegate: SpeziAppDelegate {
     override var configuration: Configuration {
         Configuration(standard: BehaviorStandard()) {
+            if HKHealthStore.isHealthDataAvailable() {
+                HealthKit {
+                    CollectSample(
+                        HKQuantityType(.stepCount),
+                        deliverySetting: .background(.afterAuthorizationAndApplicationWillLaunch)
+                    )
+                    CollectSample(
+                        HKQuantityType(.activeEnergyBurned),
+                        deliverySetting: .anchorQuery(.afterAuthorizationAndApplicationWillLaunch)
+                    )
+                }
+            }
+
```

Comment:
There is a `healthKit` section below that already contains the configuration for one HealthKit type. Please be sure that you add your additions to the section below (that is added to the main configuration in line 65-67:
```
            if HKHealthStore.isHealthDataAvailable() {
                healthKit
            }
```